File: ex_answer/ex2.java
class Dog{
    String name;
    String color;
    String age;
    String weight;

    public Dog(String name, String color, String age, String weight){
        this.name = name;
        this.color = color;
        this.age = age;
        this.weight = weight;
    }

    public void tell(){
        System.out.println("There is a " + this.color + ", " + this.age + " years old, " + this.weight + "and its name is " + this.name + ".");
    }
}

public class ex2 {
    public static void main(String args[]){
        Dog dog = new Dog("adam", "green", "20", "20");
        dog.tell(); 
    }    
}

File: ex_answer/ex2.py
class Dog:
    def __init__(self, name, color, age, weight):
        self.name = name
        self.color = color
        self.age = age
        self.weight = weight
    
    def tell(self):
        print('There is a ' + self.color + ', ' + self.age + ' years old, ' + self.weight + 'and its name is ' + self.name + '.')



dog = Dog('Adam', 'green', '13', '20')

dog.tell()